Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/EdgeSetGeometryAlgorithms.inl

```diff
@@ -701,6 +701,7 @@ typename DataTypes::Coord EdgeSetGeometryAlgorithms<DataTypes>::compute2EdgesInt
 template<class DataTypes>
 void EdgeSetGeometryAlgorithms<DataTypes>::draw(const core::visual::VisualParams* vparams)
 {
+    vparams->drawTool()->saveLastState();
```

Comment:
I see the next line is a call to PointSetGeometryAlgorithms::draw
Do you know if this save/restore LastState is using FIFO or LIFO mechanism? 
There might be a wrong behavior if the order of restore is not well handled.

Reply:
it is designed to mimic push/pop mechanism in opengl so it is FIFO

Reply:
I would have said that push-pop is LIFO :) 
Push A
Push B
Pop => returns B        (last in, first out)
Pop => returns A

Reply:
oups thats true 🫥